javascript/organizing_your_javascript_code/project_restaurant_page.md

```diff
@@ -8,61 +8,47 @@ Let's use what we've learned and take a chance to continue practicing DOM manipu
 
 #### .gitignore
 
-When working with packages that are installed with npm, you don't need to track the contents of `node_modules` with git, nor push those files to GitHub. This is because the `package.json` file contains all the information, so that anyone can clone your project and install them on their machine with `npm install`.
+When working with packages that are installed with npm, you don't need to track the contents of `node_modules` with git, nor push those files to GitHub. As we learned in the [npm lesson](https://www.theodinproject.com/lessons/node-path-javascript-npm), the `package.json` file contains all the dependency information, so that anyone can clone your project and install them on their machine with `npm install`.
 
-You can make a `.gitignore` file in the root of the project, and by writing file or directory names in it, you can tell git what things you don't want to track. It's customary to add `node_modules` to `.gitignore`, since it can get really big.
+You can make a `.gitignore` file in the root of the project, and by writing file or directory names in it, you can tell git what things you don't want to track. It's customary to add `node_modules` to `.gitignore`, since it can get really big. Similarly, `dist` is often ignored as it can be generated when someone runs the command to bundle/build the application.
 
-When creating a new repo on GitHub there is an option to specify a `.gitignore` template.  There are many templates out there that include common files and directories that are not typically tracked based on the type of project or language used.  When looking for a template for JavaScript projects there is a `node` template that includes `node_modules`.
+When creating a new repo on GitHub, there is an option to specify a `.gitignore` template.  There are many templates out there that include common files and directories that are not typically tracked based on the type of project or language used.  For JavaScript projects, there is a `node` template that includes `node_modules` andd `dist`.
```

Comment:
typo:
```suggestion
When creating a new repo on GitHub, there is an option to specify a `.gitignore` template.  There are many templates out there that include common files and directories that are not typically tracked based on the type of project or language used.  For JavaScript projects, there is a `node` template that includes `node_modules` and `dist`.
```

Have to look into why codespell caught it but is still flagging a success